### ISSUE TITLE
Can you make kalista's flee feature better?

### ISSUE THREAD

**Issue by partment** [Daeral/Marksman-AIO#1]
This addon's flee feature is awesome https://github.com/Monstertje/EloBuddy

**Comment by Daeral** [comment]
Yes, but not now, I have more important this to do at this moment. Big sorry

**Comment by partment** [comment]
Ok I will wait. By the way, when I use the combo mode Q skill didnt be used many times.
Hopes you can figure it out.

**Comment by partment** [comment]
I need to harass first and then combo.

**Comment by Daeral** [comment]
It uses Q only after attack. It's bad? I thought its supposed to be used that way

**Comment by partment** [comment]
Sometimes after engage the minions will block my Q.
So I think you can add an option for user to choose after or before.
